charts/stable/zwavejs2mqtt/values.yaml

```diff
@@ -21,6 +21,9 @@ env: {}
 service:
   port:
     port: 8091
+  # additionalPorts:
+  # ┊ - name: websocket
+  # ┊   port: 3000
```

Comment:
Are you sure you don't want to add an `additionalService` instead, would a `additionalPort` be fine for this usecase?

Reply:
Not really sure what is proper way... I copied this from deconz chart and it worked fine on my setup

Reply:
Sounds good, we can roll with it :)